test/e2e/framework/helpers/k8s/k8s_utils.go

```diff
@@ -221,6 +233,21 @@ func (k *KubeCtl) UpdatePod(pod *v1.Pod, namespace string) (*v1.Pod, error) {
 	return k.clientSet.CoreV1().Pods(namespace).Update(context.TODO(), pod, metav1.UpdateOptions{})
 }
 
+func (k *KubeCtl) PatchPod(pod *v1.Pod, namespace string, patch []map[string]interface{}, subresources ...string) (*v1.Pod, error) {
```

Comment:
This method can stay, but also create a more specific version to modifying vcores/memory.
Sth like:
```
func (k *KubeCtl) ModifyResourceUsage(pod *v1.Pod, newVcore int64, newMemory int64) {
   k.PatchPod(...) // call it properly
}
```